src/Tizen.NUI.Scene3D/src/internal/Interop/Interop.Light.cs

```diff
@@ -0,0 +1,50 @@
+﻿/*
+ * Copyright(c) 2023 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+namespace Tizen.NUI.Scene3D
+{
+    internal static partial class Interop
+    {
+        internal static partial class Light
+        {
+            [global::System.Runtime.InteropServices.DllImport(Libraries.Scene3D, EntryPoint = "CSharp_Dali_Light_New_SWIG_0")]
+            public static extern global::System.IntPtr LightNew();
+
+            [global::System.Runtime.InteropServices.DllImport(Libraries.Scene3D, EntryPoint = "CSharp_Dali_new_Light_SWIG_0")]
+            public static extern global::System.IntPtr NewLight();
+
```

Comment:
For me, `LightNew()` and `NewLight()` make a little confusing.
Could you explain them shortly?
Or how about changing `LightNew()` to just `New()` because the class name is already `Light`.

Reply:
Very good for me.
It was one of my confusing too, because every other binder uses same manner.
I will fix it only in this patch.
But we can make better modification for all the other class in next patch.

Reply:
OH :(
I'm so sorry other binder files are used like this...
Yes, we can make them in other patch!